### DISCUSSION TITLE
Explaining the torch-frame based graphs

### DISCUSSION THREAD

**Discussion by Shamskn** [pyg-team/pytorch-frame#463] [Q&A]
I am testing the Relbench model for a churn task in Kaggle. I have been running into some issues trying to plug it into GNN explainers, mainly stemming from the architecture difference as the model includes embedding steps with the help of torch-frame. Does someone have any recommendation on how I can modify this so I can test some GNN explainers on the relations data? Much Appreciated

**Comment by akihironitta** [comment]
Hi @Shamskn, thanks for sharing your question here. It'd be nice if you could share your script and error message with env details :)
